changelog/2843.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Patched a bug in `~plasmapy.diagnostics.charged_particle_radiography.synthetic_radiography` in which particles stopped
+before the detector were still included in synthetic radiographs.
```

Comment:
```suggestion
Patched a bug in `~plasmapy.diagnostics.charged_particle_radiography.synthetic_radiography` in which particles stopped
before reaching the detector were still included in synthetic radiographs.
```
In my first read through, my brain interpreted it as "the particles stopped before the detector was stopped", so this should help clarify that.